compiling/extract_and_compile.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Robert Beane
+
+# Parameters
+number=$1
+
+tar xfz NthPrime.tgz 
+gcc NthPrime/main.c NthPrime/nth_prime.c -o NthPrime/NthPrime
```

Comment:
I probably would have done `cd NthPrime` before lines 8 and 9 so I wouldn't need to include `NthPrime/` in front of _everything_.